### ISSUE TITLE
support dependency between objects, like nodes depends on its groups

### ISSUE THREAD

**Issue by immarvin** [xcat2/xcat-inventory#78]
during export and import, will process objects as well as their dependency objects, for  example, group objects are dependency objects for their member objects

**Comment by robin2008** [comment]
I don't think we need to export the parent objects by default,  maybe a hit is enough.
